### PR TITLE
Minor spec updates

### PR DESCRIPTION
**Pull request by alexmturner**
Mostly formatting and small prose changes

***
<a href="https://pr-preview.s3.amazonaws.com/patcg-individual-drafts/private-aggregation-api/pull/26.html" title="Last updated on Apr 19, 2023, 4:04 PM UTC (5bedb60)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/patcg-individual-drafts/private-aggregation-api/26/6a71a39...5bedb60.html" title="Last updated on Apr 19, 2023, 4:04 PM UTC (5bedb60)">Diff</a>

### CODE REVIEW

**Comment by alexmturner** [comment]
@linnan-github, could you PTAL? A lot of this is very minor changes to better align with the infra spec's [recommendations](https://infra.spec.whatwg.org/#algorithm-declaration) for declarations. Thanks!

**Comment by linnan-github** [comment]
> @linnan-github, could you PTAL? A lot of this is very minor changes to better align with the infra spec's [recommendations](https://infra.spec.whatwg.org/#algorithm-declaration) for declarations. Thanks!

LGTM!

File: spec.bs [hunk 1]
```diff
@@ -349,32 +334,30 @@ FLEDGE-specific algorithms {#fledge-specific-algorithms}
...
-To <dfn>fill in the signal value</dfn> given a {{PASignalValue}} |value| and an integer |maxAllowed|, run the following steps:
-1. [=Assert=] |value|'s {{PASignalValue/baseValue}} is a valid [=signal base value=].
-1. Let |returnValue| be the result of [=determining the numeric value=] of |value|'s {{PASignalValue/baseValue}}.
-1. If |value|'s {{PASignalValue/scale}} is set:
-    1. Let |returnValue| be the result of multiplying |value|'s {{PASignalValue/scale}} with |returnValue|.
-1. Let |returnValue| be the integer result of rounding |returnValue| to the nearest integer. If two integers are equally close, the result should be the integer closer to negative infinity.
-1. If |value|'s {{PASignalValue/offset}} is set:
-    1. Let |returnValue| be the result of adding |returnValue| to |value|'s {{PASignalValue/offset}}.
+To <dfn>fill in the signal value</dfn> given a {{PASignalValue}} |value| and an integer |maxAllowed|, perform the following steps. They return an integer.
+1. [=Assert=] |value|["{{PASignalValue/baseValue}}"] is a valid [=signal base value=].
```
Comment (linnan-github) [inline spec.bs:351]:
```suggestion
1. [=Assert=]: |value|["{{PASignalValue/baseValue}}"] is a valid [=signal base value=].
```

File: spec.bs [hunk 2]
```diff
@@ -286,57 +293,35 @@ To <dfn for="PrivateAggregation">obtain an aggregatable report</dfn> given an [=
...
 Issue: Fill in
 
-The <dfn algorithm for="PrivateAggregation">report scheduling steps</dfn> given an [=origin=] |reportingOrigin|, a [=worklet type identifier=] |api| and a [=list=] of {{PAHistogramContribution}} |contributions| are as follows:
+To perform the <dfn algorithm for="PrivateAggregation">report scheduling steps</dfn> with an [=origin=] |reportingOrigin|, a [=context type=] |api| and a [=list=] of {{PAHistogramContribution}} |contributions|:
 1. [=Assert=]: |reportingOrigin| is a [=potentially trustworthy origin=].
 1. If the user agent's [=aggregatable report cache=] [=set/size=] is larger than [=max stored reports=], return.
 1. Let |truncatedContributions| be a new [=list/is empty|empty=] [=list=].
 1. If |contributions| has a [=list/size=] greater than [=max report contributions=]:
-    1. [=set/For each=] |n| of [=the range=] 0 to [=max report contributions=], exclusive:
+    1. [=set/For each=] |n| of [=the exclusive range|the range=] 0 to [=max report contributions=], exclusive:
```
[the remark this answers is not in the document]
Comment (linnan-github) [reply spec.bs:305]: Actually is this a bit confusing as 0 is inclusive but max report contributions is exclusive?

File: spec.bs (hunk 1 above)
Comment (alexmturner) [reply spec.bs]: Done

File: spec.bs (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexmturner) [reply spec.bs:305]: Agree the scope of "exclusive" is a bit confusing, but this is the wording recommended by the infra spec (https://infra.spec.whatwg.org/#the-exclusive-range)

**Comment by alexmturner** [comment]
Thanks!

**Pull request merged**